backend/common/src/log_level.rs

```diff
@@ -0,0 +1,37 @@
+/// A log level that can be decoded from a string (and so can be used
```

Comment:
I don't get this, can't you just use https://docs.rs/log/0.4.14/log/enum.LevelFilter.html#impl-FromStr instead of this type?

Reply:
Good question! I don't see why not offhand :)